README.md

```diff
@@ -64,9 +64,11 @@ present, `mysqld` is executed and will run as PID 1. You can stop the
 detached container by running `docker stop <CONTAINER ID>`.
 
 ### MySQL root user
-The root user has no password set by default. You can set it by setting
-`MYSQL_ROOT_PASSWORD` environment variable when initializing your
-database.
+The root user has no password set by default, only allowing local
+connections. You can set it by setting `MYSQL_ROOT_PASSWORD` environment
```

Comment:
We've had a discussion about MySQL using the root user account, which is created by default and agreed by SCL people. Admin account wouldn't be the same as root, where root is more popular and also used by the official MySQL docs.

I don't see how consistency with mongo is relevant. Those are different worlds, most likely different users. And MYSQL_ROOT_PASSWORD is optional, mind you. If someone needs it, they should know what they are doing and they should read the docs where they'll discover that it's ROOT, not ADMIN.